internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,391 @@
+}
+
+type MessageRef struct {
+	Ref string `yaml:"$ref"`
+}
+type Operation struct {
+	OperationID string                        `yaml:"operationId"`
+	TopicTags   []spec2.Tag                   `yaml:"tags"`
+	OpBindings  spec2.OperationBindingsObject `yaml:"bindings"`
+	Message     MessageRef                    `yaml:"message"`
+}
+
+type Topic struct {
+	Publish               Operation     `yaml:"publish"`
+	Subscribe             Operation     `yaml:"subscribe"`
+	XMessageCompatibility string        `yaml:"x-messageCompatibility"`
+	TopicBinding          TopicBindings `yaml:"bindings"`
+}
+
+type TopicBindings struct {
+	Kafka KafkaBinding `yaml:"kafka"`
+}
+
+type ConfluentBrokerXConfigs struct {
+	SaslMechanisms   string `yaml:"sasl.mechanisms"`
+	SaslPassword     string `yaml:"sasl.password"`
+	SaslUsername     string `yaml:"sasl.username"`
+	SecurityProtocol string `yaml:"security.protocol"`
+}
+type ConfluentBroker struct {
+	Type     string                  `yaml:"type"`
+	XConfigs ConfluentBrokerXConfigs `yaml:"x-configs"`
+}
+
+type ConfluentSRXConfigs struct {
+	BasicAuthUserInfo string `yaml:"basic.auth.user.info"`
+}
+type ConfluentSchemaRegistry struct {
+	Type     string              `yaml:"type"`
+	XConfigs ConfluentSRXConfigs `yaml:"x-configs"`
+}
+
+var topicExists = false
+var topicNotCreated = false
+
+func newImportCommand(prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "import",
+		Short:   "Adds topics, schemas and tags from the yaml file.",
+		Example: "confluent asyncapi import spec.yaml",
+	}
+	c := &command{AuthenticatedStateFlagCommand: pcmd.NewAuthenticatedStateFlagCommand(cmd, prerunner)}
+	c.RunE = c.reverse
+	c.Flags().Bool("overwrite", false, "overwrite existing topics and schemas")
+	c.Flags().String("kafka-api-key", "", "Kafka cluster API key.")
+	c.Flags().String("schema-registry-api-key", "", "API key for Schema Registry.")
+	c.Flags().String("schema-registry-api-secret", "", "API secret for Schema Registry.")
+	return c.Command
+}
+
+func getFlagsReverse(cmd *cobra.Command) (*flagsReverse, error) {
+	overwrite, err := cmd.Flags().GetBool("overwrite")
+	if err != nil {
+		return nil, err
+	}
+	kafkaApiKey, err := cmd.Flags().GetString("kafka-api-key")
+	if err != nil {
+		return nil, err
+	}
+	schemaRegistryApiKey, err := cmd.Flags().GetString("schema-registry-api-key")
+	if err != nil {
+		return nil, err
+	}
+	schemaRegistryApiSecret, err := cmd.Flags().GetString("schema-registry-api-secret")
+	if err != nil {
+		return nil, err
+	}
+
+	return &flagsReverse{
+		overwrite:               overwrite,
+		kafkaApiKey:             kafkaApiKey,
+		schemaRegistryApiKey:    schemaRegistryApiKey,
+		schemaRegistryApiSecret: schemaRegistryApiSecret,
+	}, nil
+}
+
+func (c *command) reverse(cmd *cobra.Command, strings []string) error {
+	if len(os.Args) < 4 {
+		err := errors.New("Provide Input spec filename as argument.")
+		return err
+	}
```

Comment:
Our CLI library will do this for you. Take a look at the `ExactArgs()` function.